includes/admin/class-wcmp-export-consignments.php

```diff
@@ -118,6 +118,27 @@ private function getPickupTypeByDeliveryOptions(DeliveryOptions $delivery_option
         return AbstractConsignment::DELIVERY_TYPES_NAMES_IDS_MAP[$delivery_options->getDeliveryType() ?? AbstractConsignment::DELIVERY_TYPE_STANDARD_NAME];
     }
 
+    /**
+     * Get date in YYYY-MM-DD HH:MM:SS format
+     *
+     * @param string|null $date
+     *
+     * @return string|null
+     */
+    public function convertDeliveryDate(?string $date): ?string
```

Comment:
Hoezo mag dit null returnen? `date()` returnt een string of `false` https://www.php.net/manual/en/function.date.php

Reply:
False zou hij nooit kunnen returnen omdat anders niet door de if komt.

Reply:
done